BUILD

```diff
@@ -0,0 +1,11 @@
+objc_library(
+  name = "lib",
+  hdrs = glob(["Classes/**/*.h"]),
+  srcs = glob(["Classes/**/*.m"]),
+  includes = [
+    "Classes",
```

Comment:
Kind of too bad that `includes` isn't generated from what the `hdrs` actually resolves.  Seems like it could get out of sync.  (for my future reference: https://bazel.build/versions/master/docs/be/objective-c.html#objc_library)